womtool/src/test/resources/inputs/wdl_draft3/singlesample_v1/all.inputs.json

```diff
@@ -0,0 +1,42 @@
+{
+  "PairedEndSingleSampleWorkflow.ValidateReadGroupSamFile.ignore": "Array[String]?",
+  "PairedEndSingleSampleWorkflow.flowcell_unmapped_bams": "Array[File]",
+  "PairedEndSingleSampleWorkflow.agg_small_disk": "Int",
+  "PairedEndSingleSampleWorkflow.known_indels_sites_VCFs": "Array[File]",
+  "PairedEndSingleSampleWorkflow.wgs_evaluation_interval_list": "File",
+  "PairedEndSingleSampleWorkflow.contamination_sites_vcf": "File",
+  "PairedEndSingleSampleWorkflow.ValidateAggregatedSamFile.ignore": "Array[String]?",
+  "bwa_commandline": "String (default = StringLiteral(bwa mem -K 100000000 -p -v 3 -t 16 -Y $bash_ref_fasta))",
```

Comment:
Once #3601 merges, I should probably use the WdlWriter to override WDL 1.0's expression display string